politeiawww/api/cms/v1/v1.go

```diff
@@ -37,3 +55,44 @@ type RegisterUser struct {
 
 // RegisterUserReply replies to Register with no properties, if successful.
 type RegisterUserReply struct{}
+
+// NewInvoice attempts to submit a new invoice.
+type NewInvoice struct {
+	Month     uint16     `json:"month"`
+	Year      uint16     `json:"year"`
+	Files     []www.File `json:"files"`     // Invoice file and any attachments along with it
+	PublicKey string     `json:"publickey"` // Key used to verify signature
+	Signature string     `json:"signature"` // Signature of file hash
+}
+
+// NewInvoiceReply is used to reply to the NewInvoiceReply command.
+type NewInvoiceReply struct {
+	CensorshipRecord www.CensorshipRecord `json:"censorshiprecord"`
+}
+
+// InvoiceRecord is an entire invoice and its content.
+type InvoiceRecord struct {
```

Comment:
If I am reading this right we are excluding several things from the merkle root and signature. I don't think we want that.

Reply:
This should have been resolved by switching to JSON.